src/unstract/sdk/adapters/x2text/no_op_x2text/src/no_op_x2text_adapter.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import os
+import time
+from typing import Any, Optional
+
+from unstract.sdk.adapters.x2text.dto import TextExtractionResult
+from unstract.sdk.adapters.x2text.x2text_adapter import X2TextAdapter
+
+logger = logging.getLogger(__name__)
+
+
+class NoOpX2TextAdapter(X2TextAdapter):
+    def __init__(self, settings: dict[str, Any]):
+        super().__init__("NoOpX2Text")
+        self.config = settings
+
+    @staticmethod
+    def get_id() -> str:
+        return "noOpX2text|mp66d1op-7100-d340-9101-846fc7115676"
+
+    @staticmethod
+    def get_name() -> str:
+        return "No Op X2Text"
+
+    @staticmethod
+    def get_description() -> str:
+        return "No Op X2Text Adapter"
+
+    @staticmethod
+    def get_icon() -> str:
+        return "/icons/adapter-icons/noOpx2Text.png"
+
+    @staticmethod
+    def get_json_schema() -> str:
+        f = open(f"{os.path.dirname(__file__)}/static/json_schema.json")
+        schema = f.read()
+        f.close()
+        return schema
+
+    def process(
+        self,
+        input_file_path: str,
+        output_file_path: Optional[str] = None,
+        **kwargs: dict[Any, Any],
+    ) -> TextExtractionResult:
+        extracted_text: str = (
+            "This is a No Op x2text adapter response."
+            " This is a sample response and intended for testing \f"
+        )
+        time.sleep(self.config.get("wait_time"))
+        return TextExtractionResult(extracted_text=extracted_text)
```

Comment:
Pls make sure that choosing not to write to output_file_path etc will not cause any logical issues in any flow.

Reply:
@gaya3-zipstack Preferred not to write to file since didn't want to junk the storage with the sample response text file. 
The only place where we use it is when we switch to `Raw view` in document viewer. When NoOp x2text is chosen, it displays a message of "File not found", it is explanatory when a NoOp is used.
@hari-kuriakose Can you please help here?

Reply:
@gaya3-zipstack Like we discussed, this will be needed when summary is enabled. To retain the flow, we might need to write to the path. Making the change, thanks for pointing it out.

Reply:
@gaya3-zipstack Like we discussed, this will be needed when summary is enabled. To retain the flow, we might need to write to the path. Making the change, thanks for pointing it out.